### PR TITLE
Better menuItem visual queue for delete action

### PR DESCRIPTION
**Pull request by Octoton**
All menuItem that delete something are red for better indication:
Screenshot_20210818-163551
Screenshot_20210818-163545
Screenshot_20210818-163537
Screenshot_20210818-163519

### CODE REVIEW

File: app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    public static void setDeleteMenuItemRed(Menu menu, Context context) {
+        // All delete element inside of menu should have red text to better differentiate them
+        MenuItem liveItem = menu.findItem(R.id.action_delete_playlist);
+        if (liveItem == null)
+            liveItem = menu.findItem(R.id.action_delete_from_device);
+
+        if (liveItem != null) {
+            SpannableString s = new SpannableString(liveItem.getTitle().toString());
+            int color = context.getResources().getColor(R.color.delete);
```
Comment (AdrienPoupa) [inline app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java:23]: Looks like this call is deprecated: https://stackoverflow.com/a/31590927

File: app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+public class MenuHelper {
+
+    public static void setDeleteMenuItemRed(Menu menu, Context context) {
+        // All delete element inside of menu should have red text to better differentiate them
+        MenuItem liveItem = menu.findItem(R.id.action_delete_playlist);
+        if (liveItem == null)
+            liveItem = menu.findItem(R.id.action_delete_from_device);
+
+        if (liveItem != null) {
```
Comment (AdrienPoupa) [inline app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java:21]: At this point, liveItem is not null anymore due to the previous line, right? or could findItem return null?

Comment (Octoton) [reply app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java:21]: depending on the menu, the delete action as different naming, thus previous line can return nothing

File: app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java (hunk 1 above)
Comment (Octoton) [reply app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java:23]: will change this

Comment (Octoton) [reply app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java]: done

**Comment by soncaokim** [comment]
How this will be if the theme changes? Remaining red irregardless of theme would be inconsistent --> I would delegate the color choice to the theme.

And if we go in this direction of highlighting actions, where should be the limit? Ie. having too many colors would be counter intuitive

**Comment by Octoton** [comment]
@soncaokim I was thinking to only highlight delete action to ensure user now at a glance to be careful with this action (as it can remove permanently some stuff).
As red is the universal color for danger, it seems like a good choice for the three theme the app has. In fact, I don't really see what color other than red should be use on light and black theme.

**Comment by AdrienPoupa** [comment]
Looks good to me thanks!

**Review by AdrienPoupa — APPROVED** [review]

**Comment by soncaokim** [comment]
> @soncaokim I was thinking to only highlight delete action to ensure user now at a glance to be careful with this action (as it can remove permanently some stuff).
> As red is the universal color for danger, it seems like a good choice for the three theme the app has. In fact, I don't really see what color other than red should be use on light and black theme.

I understand your intention to warn user about destructive actions, and the use of the red makes sense somehow. 

My point is that the color (or shade of color) should/could be chosen according to the theme (bright red light theme, darker red on dark theme).

**Comment by Octoton** [comment]
@soncaokim done

**Comment by soncaokim** [comment]
I've just notices that the list's multi-select actions (CAB) are not having the same red color as the menu item.

i.e the "Delete from device..." action here-under 

Screenshot_20211003-153743~2

@Octoton Can you please have a look please?

**Pull request merged**